changelog.md

```diff
@@ -1,3 +1,109 @@
+ <a name="4.2.0"></a>
+# 4.2.0 (2019-03-06)
+[Full Changelog](https://github.com/googlechrome/lighthouse/compare/v4.1.0...v4.2.0)
+
+## New Audits
+
+* fid: add Max Potential FID to JSON ([#5842](https://github.com/googlechrome/lighthouse/pull/5842))
```

Comment:
maybe move this down into Core since we aren't exposing in the HTML report, we don't want it to show up as a headline feature